app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -61,10 +62,15 @@ const val DEFAULT_KEY_RADIUS = 0
 data class AppSettings(
     @PrimaryKey(autoGenerate = true) val id: Int,
     @ColumnInfo(
-        name = "key_size",
-        defaultValue = DEFAULT_KEY_SIZE.toString(),
+        name = "key_height",
```

Comment:
Don't change the names of existing columns, or you'll break installs.

You'll need to do a migration to add more.